src/pages/workspaces/workspace/Dashboard/CloudInformation.ts

```diff
@@ -74,9 +74,52 @@ const AzureCloudInformation = (props: AzureCloudInformationProps): ReactNode =>
 };
 
 const GoogleCloudInformation = (props: GoogleCloudInformationProps): ReactNode => {
-  const { workspace, storageDetails, storageCost, bucketSize } = props;
+  const { workspace, storageDetails } = props;
   const { accessLevel } = workspace;
-  const { googleProject, bucketName } = workspace.workspace;
+  const { bucketName, googleProject } = workspace.workspace;
+
+  const signal = useCancellation();
+
+  const [storageCost, setStorageCost] = useState<{ isSuccess: boolean; estimate: string; lastUpdated?: string }>();
```

Comment:
I can't seem to find a workspace on dev with a non-zero estimated storage cost. Do you know of one?

Reply:
I don't. 
I didn't change anything around how the value was displayed, and the loading behavior looks the same when I tested it, so I didn't worry too much about that.